src/compile_yara_signatures.py

```diff
@@ -18,12 +18,12 @@
 '''
 
 import os
+import subprocess
 import sys
-from subprocess import CalledProcessError
+from subprocess import PIPE, STDOUT, CalledProcessError
```

Comment:
Was importing `PIPE, STDOUT, CalledProcessError` directly but calling run as `subprocess.run` intentional?

Reply:
Yes, I think its more readable to use `subprocess.run` instead of `run` only.